src/sirhelpers.c

```diff
@@ -314,25 +314,28 @@ int _sir_fopen(FILE* restrict* restrict streamptr, const char* restrict filename
 
 struct tm* _sir_localtime(const time_t* restrict timer, struct tm* restrict buf) {
     if (_sir_validptr(timer) && _sir_validptr(buf)) {
-#if defined(__HAVE_STDC_SECURE_OR_EXT1__)
+#if defined(__HAVE_STDC_SECURE_OR_EXT1__) && !defined(__EMBARCADEROC__)
 # if defined(__WIN__)
```

Comment:
& I hate to be a pain in the ass, but can  you switch the order of the preproc blocks? We consistently do if !defined(__WIN__) first, since the majority of the time that will be the branch it takes

Reply:
No prob